packages/blade/src/components/Modal/ModalBody.web.tsx

```diff
@@ -0,0 +1,107 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+import React, { useCallback, useEffect, useState } from 'react';
+import styled from 'styled-components';
+import { modalBodyPadding, scrollOverlayHeight } from './modalTokens';
+import { useModalContext } from './ModalContext';
+import BaseBox from '~components/Box/BaseBox';
+import { assignWithoutSideEffects } from '~src/utils/assignWithoutSideEffects';
+import { MetaConstants, makeMotionTime, makeSize, metaAttribute } from '~utils';
+
+type ModalBodyProps = {
+  children: React.ReactNode;
+};
+
+const OverflowOverlay = styled(BaseBox)<{
+  scrollbarWidth: number;
+  footerHeight: number;
+  showOverlay: boolean;
+}>`
+  position: absolute;
+  left: 0%;
+  right: ${({ scrollbarWidth }) => `${scrollbarWidth}px`};
+  bottom: ${({ footerHeight }) => `${footerHeight}px`};
+  background: linear-gradient(
```

Comment:
Can you also add a comment that there are no tokens for the colors in this?

Reply:
We will need to tokenize this to ensure it works with multiple themes. This is pending discussion which we will close today.